client/blocks/reader-post-card/byline.jsx

```diff
@@ -79,7 +80,7 @@ class PostByline extends React.Component {
 							author={ post.author }
 							siteUrl={ streamUrl }
 							post={ post }>
-							{ post.author.name }
+							{ decodeEntities( post.author.name ) }
```

Comment:
So ... 

We should probably fix this on ingest, not up here. Look for entities in the post author and decode them during feed parsing.